routes/api.php

```diff
@@ -28,11 +28,12 @@
     Route::get('/bookings/{booking}', [App\Http\Controllers\API\BookingController::class, 'show'])->name('api.booking.show');
 
     Route::get('/roles', [App\Http\Controllers\API\RolesController::class, 'index'])->name('api.roles.index');
-    Route::get('/students', [App\Http\Controllers\API\StudentsController::class, 'index'])->name('api.students.index');
```

Comment:
It's up to you, as I don't know how you handle versioning in the API: this is a breaking change.

Reply:
It's not a breaking change because `/students` was introduced and removed before this upcoming release. Ergo: It was never released.